README.md

```diff
@@ -29,6 +29,70 @@ COUCHDB_DBS=<dbs-to-sync> # space separated list of databases you want to sync e
 npm install
 npm run local
 ```
+#### Run end-to-end test locally
+1. Update the following environment variables in your `.env` file:
+
+```
+# project wide: optional
+COMPOSE_PROJECT_NAME=pipeline
+
+# postgrest and pogresql: required environment variables for 'gamma', prod and 'local'
+POSTGRES_USER=<your-postgres-user>
+POSTGRES_PASSWORD=<your-postgres-password>
+POSTGRES_DB=<your-database>
+POSTGRES_TABLE=<your-postgres-table>
+POSTGRES_SCHEMA=<your-base-postgres-schema>
+
+# dbt: required environment variables for 'gamma', 'prod' and 'local'
+DBT_POSTGRES_USER=<your-postgres-dbt-user>
+DBT_POSTGRES_PASSWORD=<your-postgres-password>
+DBT_POSTGRES_SCHEMA=<your-dbt-postgres-schema>
+DBT_POSTGRES_HOST=<your-postgres-host> # IP address
+CHT_PIPELINE_BRANCH_URL="https://github.com/medic/cht-pipeline.git#main"
+
+# couchdb and logstash: required environment variables for 'gamma', 'prod' and 'local'
+COUCHDB_USER=<your-couchdb-user>
+COUCHDB_PASSWORD=<your-couchdb-password>
+COUCHDB_DBS=<dbs-to-sync> # space separated list of databases you want to sync e.g "medic medic_sentinel"
+COUCHDB_HOST=<your-couchdb-host>
+COUCHDB_PORT=<your-couchdb-port>
+COUCHDB_SECURE=false
+
+# superset: required environment variables for 'gamma', 'prod' and 'local'
+SUPERSET_PASSWORD=<your-superset-password>
+SUPERSET_ADMIN_EMAIL=<your-superset-emaild>
+```
+
+2. Install and run locally
+
+```sh
+# starts: logstash, superset, postgres, postgrest,  data-generator, couchdb and dbt
+npm install
+npm run local
+```
+
+3. Wait for every container to be up and running.
+4. Run end-to-end test
+
+```sh
+# runs tests/e2e-test.spec.ts
+npm test
+```
+
+5. Expected result:
+
+```sh
+ PASS  tests/e2e-test.spec.ts
+  Main workflow Test Suite
+    ✓ should have data in postgres main tables (66 ms)
+    ✓ should be able to login to superset dashboard (144 ms)
+
+Test Suites: 1 passed, 1 total
```

Comment:
This section must be updated as we add more tests with every PR, and I am afraid that we will forget. 😅  Can we mention just something like:
`Expected result - all tests passed.`

Or remove it, as the person running them will figure out themselves by the logs that the tests are failing or passing?